src/integration/asset/wit/WitAssetIntegration.ts

```diff
@@ -120,8 +117,30 @@ export default class WitAssetIntegration extends AssetIntegration<AssetSetting>
         }),
       `Time out error (5s) when getting the token with the connection URL '${this.connection.witConnection.authenticationUrl}/token'`
     );
-    // Return the Token
-    return response.data.access_token;
+    const data = response.data;
+    const expireTime = moment().add(data.expires_in, 'seconds').toDate();
+    const token : AssetConnectionTokenSetting = {
+      accessToken: data.access_token,
```

Comment:
Access token is sensitive data and should be encrypted like in Setting Service endpoint (check handleUpdateSetting)
Maybe a SettingHelper should be created to save and read settings with sensitiveData properly handled in both Service endpoint and whenever someone want to save settings.